soc/arm/nxp_imx/rt5xx/Kconfig.defconfig.series

```diff
@@ -12,7 +12,7 @@ config ROM_START_OFFSET
 	default 0x1200 if NXP_IMX_RT5XX_BOOT_HEADER
 
 config NUM_IRQS
-	default 74
+	default 76
```

Comment:
the 595 RM says 74 interrupts, so where do these extra 2 come from?

Reply:
Hi @decsny ,
The [PVT Sensor ](https://www.nxp.com/design/design-center/software/embedded-software/application-software-packs/application-software-pack-dynamic-voltage-scaling-using-pvt-sensor:APP-SW-PACK-DVS-PVT-SENSOR) is a special feature in the RT500 that can reduce dynamic power consumption.  We are using this feature in [NXP's Zephyr SmartWatch demo](https://github.com/nxp-zephyr/apps_zephyr/tree/SmartWatch).  And the PVT Sensor uses [IRQ 75](https://github.com/nxp-appcodehub/ap-dvs-pvt-sensor/blob/fc62a7895ff3a590b11df6791fe398ace23216f3/dvs_pvt_sensor/app/evkmimxrt595/pvt_lib/pvt_int.h#L17).  But you are correct, that is not documented in the Reference Manual.  Best regards

Reply:
If it isn't documented, should we put a comment in this file highlighting this for future reference?

Reply:
Thanks @dleach02 and @decsny ,
I added a comment to explain this change.